### DISCUSSION TITLE
The cf1 does not appear on RViz with motive 1.9

### DISCUSSION THREAD

**Discussion by Carmol1** [USC-ACTLab/crazyswarm#651] [Q&A] [answered]
Hello.

I am using the next cofiguration in hover_swarm.launch.

motion_capture_type: "optitrack_closed_source " # one of none,vicon,optitrack,optitrack_closed_source,qualisys,vrpn
      object_tracking_type: "motionCapture" # one of motionCapture,libobjecttracker
      send_position_only: False # set to False to send position+orientation; set to True to send position only
      motion_capture_host_name: "192.168.0.100"
      motion_capture_interface_ip: "192.168.0.100" # optional for optitrack with multiple interfaces
      save_point_clouds: "/dev/null" # set to a valid path to log mocap point cloud binary file.
      print_latency: False
      write_csvs: False
      force_no_cache: False
      enable_parameters: True
      enable_logging: False
      enable_logging_pose: True

In the file crazyflies.yaml I have put

crazyflies:

channel: 80
id: 1
initialPosition: [1.5, 1.5, 0.0]
type: default

When using roslaunch crazyswarm hover_swarm.launch, in the last lines I get the following 

[ INFO] [1640171088.998566791]: [cf1] CONTROLLER: Using Mellinger (2) controller
[ INFO] [1640171089.009248978]: Update params: 0.092075 s
[ INFO] [1640171089.009606891]: Started 1 threads

and RViz appears, but it does not shows the cf1 and the option global status is yellow and it shows no tf data, on the other hand, if I use python3 hello_world.py the carzyflie takes off and lands, this being the expected, for this reason I do not know what's wrong.

I would appreciate any tip to resolve this.

**Comment by whoenig** [comment, marked as answer]
Do you use the flow deck? The missing visualization in rviz indicates that there is a problem with receiving the tracking information from optitrack. On the other hand, the successful flight indicates that your CF has enough state information for a short flight. A good test case would be to check if the mocap_helper.launch file receives data (it will print to the console the data it receives from optitrack).

**Reply by Carmol1** [reply]
Hello.

Thank you for responding, I don't know what flow deck is, on the other hand, I was able to solve the problem by changing in motive multicast to unicast, so far cf1 already appears in RViz and by updating crazyradio pa to the version 99.55 and executing hello_world. p the crazyflie takes off and lands correctly, it is worth mentioning that by pressing the correct buttons on the control the crazyflie also takes off and lands, however my question is if later on can I have problems using unicast instead of multicast?

Greetings.

**Reply by whoenig** [reply]
Unicast should be fine, especially if you only use one PC that receives the data from Motive.

**Reply by Carmol1** [reply]
Great, problem solved, thank you very much for your help.
